auth-web/src/components/auth/create-account/PaymentMethodSelector.vue

```diff
@@ -88,8 +103,13 @@ export default class PaymentMethodSelector extends Mixins(Steppable) {
   }
 
   private get isEnableCreateBtn () {
-    return (this.selectedPaymentMethod === PaymentTypes.PAD)
-      ? (this.selectedPaymentMethod && this.isPADValid) : !!this.selectedPaymentMethod
+    if (this.selectedPaymentMethod === PaymentTypes.PAD) {
+      return this.isPADValid
+    } else if (this.selectedPaymentMethod === PaymentTypes.BCOL) {
+      return !(this.currentOrganization.bcolProfile?.password === undefined || this.currentOrganization.bcolProfile?.password === '')
```

Comment:
I'm not a huge fan of using emits, I think we should be using state instead throughout the code